### ISSUE TITLE
query file compression

### ISSUE THREAD

**Issue by fatimaaysha** [mathjeff/Mapper#1]
Is it possible for me to have .fastq.gz compressed files as my queries?

**Comment by mathjeff** [comment]
Yes! Thanks for asking. I've now updated the README to give more info about the usage. Does it work for you?
